MinecraftClient/Protocol/Handlers/Protocol18Terrain.cs

```diff
@@ -79,45 +80,97 @@ public void ProcessChunkColumnData(int chunkX, int chunkZ, ushort chunkMask, ush
 
                         Chunk chunk = new Chunk();
 
-                        if (dataArray.Length > 0)
+                        if (protocolversion >= Protocol18Handler.MC116Version)
```

Comment:
I see no changes for 1.16 in https://wiki.vg/Chunk_Format#Chunk_Section_structure ; so I'm not 100% sure what is done here specifically for 1.16. If it's a more optimized chunk parsing routine, it should work also below 1.16 so no need to duplicate the code here, right?

Reply:
No. The document for chunk format is outdated and doesn't work on 1.16.

Reply:
Thanks for pointing that out, everything makes sense 👍
I adjusted the parsing code to use your approach instead and disable the padding part when parsing 1.15 chunk data.